lib/prototype-admin/reset-done.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Data reset - NHS prototype kit
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+      <h1 class="nhsuk-heading-l">Data reset</h1>
+
+      <p class="nhsuk-body">The session data has been reset.</p>
+
+      <p><a href="{{ returnPath }}" class="nhsuk-link">Return</a>
```

Comment:
Does this one need `nhsuk-body` like the paragraph above?

```suggestion
      <p class="nhsuk-body"><a href="{{ returnPath }}" class="nhsuk-link">Return</a>
```

Reply:
I think the style is applied by default to `<p>`, and same with `nhsuk-link` (which actually does nothing at all), but no harm in adding them in case this changes in future.